_books/ion-1-1/src/modules/system_module.md

```diff
@@ -39,18 +51,19 @@ The Ion 1.1 System Symbol table _replaces_ rather than extends the Ion 1.0 Syste
 | 14 | `macro_table`                   |
 | 15 | `symbol_table`                  |
 | 16 | `module`                        |
-| 17 | `retain`                        |
-| 18 | `export`                        |
-| 19 | `catalog_key`                   |
+| 17 | ~~retain~~                      |
```

Comment:
I think leaving these in the table doesn't buy us anything unless we're also going to have a note explaining why they were removed. That feels more like a github issue to open/close though, since the spec itself doesn't need to contain its own history.